Makefile

```diff
@@ -104,7 +104,7 @@ kops-arm-ubuntu-22: kops-prereqs
 	RELEASE=$(RELEASE) $(KOPS_ENTRYPOINT);
 
 .PHONY: kops-prereqs
-kops-prereqs: postsubmit-build
+kops-prereqs: $(if $(JOB_TYPE) == "presubmit",,postsubmit-build)
```

Comment:
this is not quite right...  you probably want this

`$(if $(filter presubmit,$(JOB_TYPE)),,postsubmit-build)`

Check Common.mk youll see this kind of thing.

Reply:
Used [open_pr_check.sh](https://github.com/aws/eks-distro-build-tooling/blob/64e9741d152acddf6b6aa54cdbae77ac728232be/pr-scripts/open_pr_check.sh#L26) as the example. Will update to your recommendation. Don't see us doing this in eks-distro, other than a single call in the [buildkit.sh](https://github.com/aws/eks-distro-build-tooling/blob/64e9741d152acddf6b6aa54cdbae77ac728232be/scripts/buildkit.sh#L92) where we trim, but not sure why this is the only example of us trimming in that repo.